### PR TITLE
edited a way to get  at AlbertAttention.forward

### PR DESCRIPTION
**Pull request by jinkilee**
'''
# Should find a better way to do this
w = (
    self.dense.weight.t()
    .view(self.num_attention_heads, self.attention_head_size, self.hidden_size)
    .to(context_layer.dtype)
)
b = self.dense.bias.to(context_layer.dtype)
'''

I thought the above code is not necessary.
it can be simply fixed by "merging" `context_layer` at forward().
I committed my "merging" function at AlbertAttention

'''
def merge_tensor(self, x):
    s = x.size()[-2]
    return torch.cat([x[:,:,i,:] for i in range(s)], dim=-1)
'''

I wanted to make a test by "make test' as described in CONTRIBUTING.md
but I couldn't because i faced some make error.

This is my first open source contribution. If i forgot something, please let me know, so I can fix and follow up.

### CODE REVIEW

**Comment by jinkilee** [comment]
will commit with another pull request
